Jenkinsfile

```diff
@@ -95,10 +95,11 @@ pipeline {
                         )
                     ]) {
                         script {
-                            // we still want to run the platform deploy even if this fails, hence try-catch
+                            // we still want to run the deploy even if this fails, hence try-catch
                             try {
                                 sh 'git reset --hard HEAD'
                                 sh 'git pull --tags'
+                                sh 'npm run build'
```

Comment:
this shouldn't be necessary, given line 62. build and publish steps shouldn't be combined, and we must figure out why you'd have to build it again... but i have a feeling this won't fix it.